Icon and Diamond shape fixed ✅ for web support 🔥

mahmoud-eslami opened the pull request:
**Hi there,**
I tested your package on mobile, and everything worked fine.
But there was some issue with the `icon` and `diamond` shape on the web version .
Please check the below screenshot :

<img width="400" alt="issue" src="https://user-images.githubusercontent.com/57481226/144919449-7c113609-32c3-43a5-aba3-aeb17d274710.png">

I added some changes on the source code. Result :

<img width="200" alt="result1" src="https://user-images.githubusercontent.com/57481226/144919693-5ed3a07c-c446-46a2-939f-c7450414c956.png">
<img width="400" alt="result2" src="https://user-images.githubusercontent.com/57481226/144919722-ff6c59e6-e5f5-400e-b651-ec6fc37763ba.png">

mahmoud-eslami committed:
Icon and Diamond size fixed ✅ for web support 🔥

hosain-mohamed commented:
Hi Pretty Good thanks  

But Look at this How it will look like on a phone not the web
![Simulator Screen Shot - IOS 13  - 2021-12-06 at 23 38 15](https://user-images.githubusercontent.com/16990315/144926824-467710c2-be06-485d-99a2-fb75d59991ea.png)


The change you add is great but needs a small change where you check if the width is less than 1000
then you can return what I used for the size getRelativeWidth(0.135) like this 

```double getDiamondSize() {
  var width = SizeConfig.screenWidth;
  if (width > 1000) {
    return 0.04 * SizeConfig.screenWidth;
  } else {
    return 0.135 * SizeConfig.screenWidth;
  }
}
```

please change it to this and I will merge your request 

Thanks

mahmoud-eslami committed:
Add some condition to handle diamond size.

mahmoud-eslami commented:
Thanks for the quick response. I added `0.135 * SizeConfig.screenWidth` to show the diamond in mobile device with standard size, Also added some conditions to show diamond better in other sizes.

hosain-mohamed commented:
Merged Thanks 🌹
